infrastructure/collections/src/main/java/tech/pegasys/teku/infrastructure/collections/cache/LRUCache.java

```diff
@@ -28,31 +28,27 @@
  */
 public class LRUCache<K, V> implements Cache<K, V> {
 
+  public static <K, V> LRUCache<K, V> create(int capacity) {
+    return new LRUCache<>(
+        SynchronizedLimitedMap.createEmpty(),
+        initEntries ->
+            SynchronizedLimitedMap.create(
+                capacity, LimitStrategy.DROP_OLDEST_ELEMENT, initEntries));
```

Comment:
I'd have thought we'd want to drop least recently accessed here, not oldest.  We want to keep the data we're using, regardless of when it was first requested.

Reply:
Yep, let's stick to only `DROP_LEAST_RECENTLY_ACCESSED` mode. 
Though it's `get()` performance is slightly weaker `17.5M ops/sec` v.s. `25M ops/sec`